app/api/v2/endpoints/accounts_endpoint.rb

```diff
@@ -8,7 +8,7 @@ class AccountsEndpoint < Grape::API
         end
         get ':id' do
           account = Account.find params[:id]
-          header 'Updated-At', account.updated_at.to_s
+          header 'Updated-At', account.updated_at.to_json.tr("\"", "")
```

Comment:
Oh so the presenter was returning ISO 8601, but `account.updated_at.to_s` here wasn't? Then I think you can simply do `account.updated_at.iso8601`

Reply:
Gotcha. I'm going to have to convert it to `Time`, I think, to avoid millisecond precision. The Gravity PR was quite helpful, thanks!